### DISCUSSION TITLE
Custom Links: Link Groups, Table Columns

### DISCUSSION THREAD

**Discussion by vladimirs-git** [netbox-community/netbox#14190] [Q&A] [answered]
Please show example, how to configure Custom Links: [Link Groups](https://github.com/netbox-community/netbox/blob/develop/docs/customization/custom-links.md#link-groups) and [Table Columns](https://github.com/netbox-community/netbox/blob/develop/docs/customization/custom-links.md#table-columns) options.

I expect to see dropdown links.

**Comment by zmaster7** [comment, marked as answer]
By specifying a "Group name" when creating custom links, a dropdown with that Group name will be created automatically.

example:
![image](https://github.com/netbox-community/netbox/assets/46574676/ff4fa345-251b-48b3-86cc-ce04ade91b8f)
![image](https://github.com/netbox-community/netbox/assets/46574676/39e78150-3372-445e-92b8-f9ab5cb4d5c2)

As for table columns, you can get them to show up by going to the table view for objects you created custom links for (e.g. Devices), click "Configure Table" on the top right corner, and choose your custom link field name:

![image](https://github.com/netbox-community/netbox/assets/46574676/aaac317b-ec09-4280-987d-95fd3efdebec)
![image](https://github.com/netbox-community/netbox/assets/46574676/d940a135-72c8-4bf9-bac5-21b1423b25ce)
![image](https://github.com/netbox-community/netbox/assets/46574676/41dedf87-e05e-496a-b640-25074ce6ce54)

**Comment by vladimirs-git** [comment]
Understood, thank you.
